README.EN.md

```diff
@@ -14,26 +14,54 @@ It is important to note that this code should not be used with high values becau
 
 ## How to configure the robot
 
-To configure you must change the file `config.js` as we will show below:
-
-```JavaScript
-export default {
-  apiKey: "qw8e4q6",
-  apiSecret: "d8fg9d",
-  amount: 0.015,
-  profitCurrency: "BRL",
-  initialBuy: true,
-  minProfitPercent: 0.01
-};
-```
+
+### Requirements
+
+* You'll need to install [docker](https://docs.docker.com/). To install you can follow the official [documentation](https://docs.docker.com/engine/install/ubuntu/).
+* You must have a verifed account in the [Biscoint](https://biscoint.io/quick-register)  to create the credentials to be able to interacte with the API.
+* Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if don't have it.
+
+### Git clone
+
+Clone the project running the following command:
+
+`git clone https://github.com/Biscoint/biscoint-arbitrage-bot.git`
+
+### Create the access keys
+
+Access the [interface  to obtain the keys of Biscoint](https://biscoint.io/dashboard/API). In case the menu "API" doesn't show, ask to the [support](https://biscoint.io/support) to activate the creation of the keys to your account.
+
+To configure you'll need to provide the values below using  environment variables:
+
 
 Explaining each configuration item:
 
 - **apiKey** - here you must fill in your api key generated in Biscoint
 - **apiSecret** - here you must fill in your secret key generated in Biscoint
-- **amount** - here you must enter the amount you want to arbitrate, by default you will arbitrate 0.015 BTC to win reais
-- **profitCurrency** - by default here will be BRL, that is, you will arbitrate to increase your balance in reais
-- **initialBuy** - in the example the robot always starts buying because it is set here to "true"
-- **minProfitPercent** - here is the minimum that the arbitrage must provide profit for the robot to execute it
+- **amount** - opcional - here you must enter the amount you want to arbitrate, by default you will arbitrate 0.015 BTC to win reais
```

Comment:
```suggestion
- **amount** - optional - here you must enter the amount you want to arbitrate, by default you will arbitrate 0.015 BTC to win reais
```

Reply:
I'll fix it in the comments... it happen when you do to much copy and paste lol :)